src/document/DocumentAnnotator.scss

```diff
@@ -10,9 +10,23 @@
         pointer-events: none;
     }
 
+    .textLayer {
+        pointer-events: none; // Ensures that click events are received on the create region layer
+
+        span {
+            pointer-events: auto; // Text layer still receives text highlight events
+        }
+    }
+
     &.ba-is-highlighting {
         .textLayer > span {
             @include ba-HighlightCreator-cursor;
         }
     }
+
+    &.ba-is-create-region {
```

Comment:
Does this work in both the feature flip enabled/disabled scenarios?

Reply:
Yep, when the feature flip is disabled the behavior is the same as what was on master.